### ISSUE TITLE
Get and Taildrop errors after activating exit node on Synology

### ISSUE THREAD

**Issue by truthsword** [tailscale/tailscale#3407]
### What is the issue?

Activated/tested exit node on Synology. All seemed fine.

Tried to collect tranferred files to Synology using 
`sudo tailscale file get .'

Error instead:
```
sudo tailscale file get /volume1/downloads/
getting WaitingFiles: HTTP 500 Internal Server Error: peerapi disabled
 (expected 200)
```
Next I tried to taildrop file fronm PC to Synology... but Synology was not available "target"

Thinking that exit node might be issue, I stopped exit node, and still Synology was not target for taildrop.

Stopped and restarted Synology package. Same issue. Not target.

So I cannot
1. "get" taildropped files from Synology
2.  nor, send files to Synology via taildrop

Dashboard shows Synology "Connected"

### Steps to reproduce

Tried to collect tranferred files to Synology using 
`sudo tailscale file get .'

or 

taildrop file from PC to Synology (Synology not shown on send to list)

### Are there any recent changes that introduced the issue?

Updated from 1.12 to 1.18
Turned on Exit Node on Synology

### OS

Synology

### OS version

DSM6

### Tailscale version

1.18.0-014

### Bug report

BUG-2eb464bd5ec25351d8682b763d5e6f11f1491eaa9eaf6770bc854016412232c7-20211125162231Z-577d3b1e1452d807

**Comment by bradfitz** [comment]
Yeah, we haven't made Taildrop work on Synology yet. We have a plan now (have the user create a "Taildrop" share to opt-in), but haven't yet finished it.

**Comment by DentonGentry** [comment]
Fixed in the 1.18.2 Synology release by https://github.com/tailscale/tailscale/commit/6c44133d8f1cfe06c33ac924daf425d8dc7f5fab. You can sideload it from https://github.com/tailscale/tailscale-synology/releases now, or we expect it to appear in the Package Center in (hopefully) January.
